FactorioClient.py

```diff
@@ -85,8 +91,12 @@ def on_print(self, args: dict):
 
     def on_print_json(self, args: dict):
         if self.rcon_client:
-            text = self.factorio_json_text_parser(copy.deepcopy(args["data"]))
-            self.print_to_game(text)
+            is_uninteresting_item_send: bool = args.get("type", "") == "ItemSend" \
+                                               and not self.slot_concerns_self(args["receiving"]) \
+                                               and not self.slot_concerns_self(args["item"].player)
```

Comment:
I would make that a function so it does not get evaluated if the filter is disabled. if you don't want to change, it i'm fine with merging as-as though.

As a side note, the `: bool` annotation makes this harder to read imo. It adds indentation, the "is_" should be clear to a reader and the IDE and mypy should be able to figure out the type without that annotation.